extensions/amp-story/1.0/amp-story.js

```diff
@@ -1553,10 +1553,10 @@ export class AmpStory extends AMP.BaseElement {
       maskEl.addEventListener('click', () => {
         const actions = Services.actionServiceForDoc(this.element);
         if (this.sidebar_) {
+          this.closeOpacityMask_();
```

Comment:
The side effects of `openOpacityMask_` and `closeOpacityMask_` are asynchronous, since they are wrapped in a `mutateElement`, so changing the order of these calls shouldn't really have an effect. If you need the `actions.execute` to happen after the opacity mask has opened, you will need to return the Promise from `mutateElement`

Reply:
I'm not really sure why (which is always bad, I know), but I noticed that when the order of these calls were changed the opening transition for the sidebar worked more reliably. Prior to being changed, the transition-in for the sidebar would jerk ever so often.

Reply:
If it's stable now this is fine, but if it continues to jerk, then waiting on the Promise could be a good solution at that time